manageability-and-operations/observability-and-manageability/external-database-enablement/root_module/modules/pdb/variables.tf

```diff
@@ -0,0 +1,130 @@
+#Common database system variables
+
+variable "compartment_ocid" {
+  type = string
+}
+
+variable "external_database_connector_connection_string_hostname" {
+  description = "Hostname for connection string"
+  type        = string
+}
+
+variable "external_database_connector_connection_string_port" {
+  description = "Port for connection string"
+  type        = number
+  default     = 1521
+}
+
+variable "external_database_connector_connection_string_protocol" {
+  description = "Protocol for connection string. Can be 'TCP' or 'TCPS'"
+  type        = string
+  validation {
+    condition     = contains(["TCP", "TCPS"], var.external_database_connector_connection_string_protocol)
+    error_message = "The value of 'protocol' in the JSON-input for database systems must be 'TCP' or 'TCPS'"
+  }
+}
+
+variable "ssl_secret_id" {
+  description = "The OCID for the OCI Vault Secret used for TCPS-connections. PKCS12 and JKS are supported as trust store types"
+  type        = string
+  default     = null
+}
+
+variable "external_database_connector_agent_id" {
+  description = "The OCID for the management agent used for database connections"
+  type        = string
+  #default = "ocid1.managementagent.oc1..XXXXXXX"
```

Comment:
Remove line.

Reply:
Removed placeholder OCIDs in the three variable.tf-files specified. See commit [c8b0810](https://github.com/oracle-devrel/technology-engineering/pull/856/commits/c8b0810732d9704a0e2662aaa9bbcc2b859acffb)